components/brave_vpn/resources/panel/components/settings-panel/index.tsx

```diff
@@ -49,72 +52,93 @@ function SettingsPanel(props: Props) {
     getPanelBrowserAPI().serviceHandler.enableOnDemand(checked)
   }
 
+  const handleSmartProxyRoutingChange = ({ checked }: { checked: boolean }) => {
+    setSmartProxyRoutingEnabled({ enabled: checked })
+    getPanelBrowserAPI().serviceHandler.enableSmartProxyRouting(checked)
+  }
+
   return (
-    <S.Box>
-      <S.PanelContent>
+    <Style.Box>
+      <Style.PanelContent>
         <PanelHeader
           title={getLocale('braveVpnSettingsPanelHeader')}
           buttonAriaLabel={
             getLocale('braveVpnSettingsPanelBackButtonAriaLabel')
           }
           onClick={props.closeSettingsPanel}
         />
-        <S.SettingsList>
+        <Style.SettingsList>
           {onDemand.available && (
             <>
-              <S.Setting
+              <Style.Setting
                 onClick={
                   e => handleToggleChange({ checked: !onDemand.enabled })
                 }
               >
-                <S.SettingLabel>
+                <Style.SettingLabel>
                   {getLocale('braveVpnReconnectAutomatically')}
-                </S.SettingLabel>
+                </Style.SettingLabel>
                 <Toggle
                   checked={onDemand.enabled}
                   onChange={handleToggleChange}
                   size='small'
                   aria-label='Reconnect automatically'
                 />
-              </S.Setting>
-              <S.Divider />
+              </Style.Setting>
+              <Style.Divider />
             </>
           )}
-          <S.Setting
+          <Style.Setting
+            onClick={
+              e => handleSmartProxyRoutingChange({ checked: !smartProxyRoutingEnabled.enabled })
+            }
+          >
+            <Style.SettingLabel>
+              {getLocale(S.BRAVE_VPN_SMART_PROXY_ROUTING)}
+            </Style.SettingLabel>
+            <Toggle
```

Comment:
Not sure if its worth it but the toggle supports slotting a label in, so it'll automatically be clickable:


```tsx
          <Style.Setting>
            <Toggle checked={...} onChange={...}>
              <Style.SettingLabel>
                {getLocale(S.BRAVE_VPN_SMART_PROXY_ROUTING)}
              </Style.SettingLabel>
           </Toggle>
          </Style.Setting>
```

(you'd have to set `--leo-toggle-label-flex-direction: row-reverse` on `Style.Setting` too)

Reply:
tested but it's difficult to adjust the the space between them.